core/src/main/java/com/thoughtworks/ark/core/utils/NetworkUtil.kt

```diff
@@ -0,0 +1,24 @@
+package com.thoughtworks.ark.core.utils
+
+import android.content.Context
+import android.content.Context.CONNECTIVITY_SERVICE
+import android.net.ConnectivityManager
+import android.net.NetworkCapabilities
+
+object NetworkUtil {
```

Comment:
We have a function `isNetworkReachable` in `NetworkReachableInterceptor`

Reply:
get it